### PR TITLE
Update GraphQL docs

### PR DESCRIPTION
**Pull request by p-janik**

### CODE REVIEW

File: SDK.markdown [hunk 1]
```diff
@@ -5,3 +5,203 @@ permalink: /sdk/
...
+
+const result = await client.graphQL.rawQuery(query);
+```
+
+{::comment}
+
+## Model Reference
+
+Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+{:/comment}
```
Comment (johannespfeiffer) [inline SDK.markdown]: nit: let's remove this part

File: docs.markdown [hunk 2]
```diff
@@ -240,202 +240,2624 @@ Result:
...
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
-{:/comment}
+## GraphQL Models
+
+<!-- START graphql-markdown -->
+
+# Schema Types
+
+<details>
+  <summary><strong>Table of Contents</strong></summary>
```
Comment (johannespfeiffer) [inline docs.markdown]: Can we skip the TOC?

File: SDK.markdown [hunk 3]
```diff
@@ -5,3 +5,203 @@ permalink: /sdk/
 ---
```
Comment (johannespfeiffer) [inline SDK.markdown:6]: Let's add `sidebar: true` to SDK as well.

**Review by johannespfeiffer — APPROVED** [review]
Looks good, please see the two minor suggestions.

File: SDK.markdown (hunk 3 above)
Comment (p-janik) [reply SDK.markdown:6]: nice, missed this one

File: docs.markdown (hunk 2 above)
Comment (p-janik) [reply docs.markdown]: yes, I'm working on that, looks broken

**Review by johannespfeiffer — APPROVED** [review]
Looks good, LGTM 👍

**Pull request merged**